### ISSUE TITLE
frontend - add a revoke button

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-lcp-server#134]
Add a revoke button in the list of transactions. This button should be in a different area than the "user" buttons, because it's logically a "provider" action. It will call the "status" route of the LSD server (a PATCH) with a "revoked" status in a partial lsd json structure. 
The errors must be managed and displayed on the screen (popup or message).

**Comment by llemeurfr** [comment]
pls create a feature branch for this enhancement.

**Comment by danielweck** [comment]
https://github.com/readium/readium-lcp-server/blob/develop/lsdserver/api/license_status.go#L836

```go
func decodeJsonLicenseStatus(r *http.Request, ls *licensestatuses.LicenseStatus) error {
	var dec *json.Decoder

	if ctype := r.Header["Content-Type"]; len(ctype) > 0 && ctype[0] == api.ContentType_FORM_URL_ENCODED {
		buf := bytes.NewBufferString(r.PostFormValue("data"))
		dec = json.NewDecoder(buf)
	} else {
		dec = json.NewDecoder(r.Body)
	}

	err := dec.Decode(&ls)

	return err
}
```

**Comment by danielweck** [comment]
Related issue:
https://github.com/readium/readium-lcp-server/issues/132

**Comment by danielweck** [comment]
Code paths:

https://github.com/readium/readium-lcp-server/blob/develop/lsdserver/api/license_status.go#L649
`func CancelLicenseStatus(w http.ResponseWriter, r *http.Request, s Server) {}`

https://github.com/readium/readium-lcp-server/blob/develop/lsdserver/server/server.go#L93
`s.handlePrivateFunc(licenseRoutes, "/{key}/status", apilsd.CancelLicenseStatus, basicAuth).Methods("PATCH")`

**Comment by Scarsniik** [comment]
fixed in 43e7969

**Comment by llemeurfr** [comment]
tested, ok.
